MCR-1791 update to JQuery 3.3.1

yagee-de opened the pull request.

yagee-de committed:
MCR-1791 update to JQuery 3.3.1

kkrebs reviewed (changes requested):
WebCLI can not be used with this update! Clicking on Commands-DropDown leads to en empty page

eckstueck committed:
MCR-1791 update to JQuery 3.3.1

updated Bootstrap to 3.3.7

eckstueck committed:
MCR-1791 update to JQuery 3.3.1

- updated JQuery to 3.3.1
- updated Bootstrap to 3.3.7
- updated Select2 to 3.5.1

eckstueck committed:
MCR-1791 update to JQuery 3.3.1

- updated JQuery to 3.3.1
- updated Bootstrap to 3.3.7
